superset/annotation_layers/annotations/commands/bulk_delete.py

```diff
@@ -36,8 +36,10 @@ def __init__(self, model_ids: list[int]):
 
     def run(self) -> None:
         self.validate()
+        assert self._models
```

Comment:
This logic is temporary. It _should_ be cleaned up when we refactor the commands and enforce that `validate` doesn't augment any properties.